app/src/main/java/com/linecorp/linesdktest/InternalApisFragment.java

```diff
@@ -300,17 +307,19 @@ void updateAgreementStatus() {
 
     @OnClick(R.id.openchat_create_chat_btn)
     void createChatroom() {
-        startApiAsyncTask("createChatroom", () -> {
-            OpenChatParameters parameters = new OpenChatParameters.Builder()
-                    .setName("Demo openchat room")
-                    .setDescription("This is a demo chatroom description")
-                    .setCategoryId(17)
-                    .setCreatorDisplayName("Demo app owner")
-                    .setIsSearchable(true)
-                    .build();
-
-            return openChatApiClient.createOpenChatRoom(parameters);
-        });
+        OpenChatParameters parameters = new OpenChatParameters(
+                "Demo openchat room",
+                "This is a demo chatroom description",
+                "Demo app owner",
+                OpenChatCategory.Game,
+                true);
+        startApiAsyncTask("createChatroom", () -> openChatApiClient.createOpenChatRoom(parameters));
+    }
+
+    @OnClick(R.id.openchat_create_chat_ui_btn)
+    void createChatroomWithUi() {
+        Intent intent = openChatApiClient.getCreateOpenChatRoomIntent(requireActivity());
```

Comment:
As mentioned in https://github.com/line/line-sdk-android/pull/45#discussion_r364109059, the `Intent` is supposed to be from `CreateOpenChatActivity`.
In `InternalApisFragment.kt`,
```kotlin
@OnClick(R.id.openchat_create_chat_ui_btn)
void createChatroomWithUi() {
    Intent intent = CreateOpenChatActivity.createIntent(requireActivity(), channelId);
    startActivityForResult(intent, REQUEST_CODE_CREATE_OPEN_CHATROOM);
}
```
In `CreateOpenChatActivity.kt`,
```kotlin
class CreateOpenChatActivity : AppCompatActivity() {
    private val openChatApiClient: OpenChatApiClient by lazy {
        val channelId = intent.getStringExtra(ARG_CHANNEL_ID).orEmpty()
        OpenChatApiClientImpl(this, Uri.parse(BuildConfig.API_SERVER_BASE_URI), channelId)
    }

    ...

    companion object {
        ...
        @JvmStatic
        fun createIntent(context: Context, channelId: String): Intent =
            Intent(context, CreateOpenChatActivity::class.java).apply {
                putExtra(ARG_CHANNEL_ID, channelId)
            }
        ...
    }
}

```